### ISSUE TITLE
[FEATURE] Replication of VirtualMachines across Harvester Clusters - DR

### ISSUE THREAD

**Issue by ibrokethecloud** [yasker/vm#1850]
**Is your feature request related to a problem? Please describe.**

Harvester should support replication of VMs across Harvester Clusters. 
This can be useful when looking to migrate VM's from one cluster to another, and also for DR scenarios.

**Describe the solution you'd like**

Longhorn already supports ability to backup and replication volumes across K8S clusters. Harvester needs to leverage the same to replicate the volumes and associated VirtualMachine metadata to another target cluster.

**Describe alternatives you've considered**

**Additional context**

**Comment by asettle** [comment]
Related to Harvester support for DR Volumes

**Comment by abonillabeeche** [comment]
Don't let this one die!!

**Comment by innobead** [comment]
#10419

**Comment by Vicente-Cheng** [comment]
We need to discuss #10419 and have the specific scope for the DR feature. 

cc @harvester/storage, @harvester/tpm, @ihcsim, @innobead

**Comment by forbesguthrie** [comment]
This shouldn't be blocked by a broader DR strategy.  We need to be able to deliver cross cluster/namespace replication (and therefore migration) as an important operational workflow.  DR is a very specific use-case which may utilize this, but if we tie everything required for a sane DR solution, then this will continue to delayed.
We should also consider splitting this into storage agnostic migration, and Longhorn-backed migration/replication.  The first is more broadly useful, but the second could efficiencies for those VMs.  I presume there will be different mechanisms used, but the customer workflow should be the same.

**Comment by innobead** [comment]
> This shouldn't be blocked by a broader DR strategy. We need to be able to deliver cross cluster/namespace replication (and therefore migration) as an important operational workflow. 

Yes, in fact, live migration is not identical to DR: one moves running VMs with their volumes, whereas the other is replication for an active backup.
